actions/src/issues/checkIssueAssignee.js

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Issue assigned handler.
+ */
+
+const core = require('@actions/core');
+const { context, GitHub } = require('@actions/github');
+const { google } = require('googleapis');
+
+const checkAssignees = async () => {
+  core.info('A USER GOT ASSIGNED TO AN ISSUE...');
+  const token = core.getInput('repo-token');
+  const octokit = new GitHub(token);
+  const assignee = context.payload.assignee;
+  const issue = context.payload.issue;
+  const linkToCla = 'here'.link(
+    'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up');
+
+  core.info('Checking if ' + assignee.login + ' has signed the CLA');
+  const assigneeHasSignedCla = await hasSignedCla(assignee.login);
```

Comment:
@ankita240796, this is where the error could be thrown for sheets.
I will wrap this in a try-catch block to handle the error.